pkg/amdsevsnp/report.go

```diff
@@ -131,8 +132,10 @@ func (sr *report) report() Report {
 // https://www.amd.com/system/files/TechDocs/56860.pdf
 // MSG_REPORT_RSP Table 23.
 // NOTE: reportResponse.Report is a byte slice, to have the original
-//     response in bytes. The conversion to internal struct happens inside
-//     convertRawReport.
+//
```

Comment:
ugh, maybe we should remove the leading spaces.

Reply:
opps, autoformat
fixing